### PR TITLE
Add links to titles

### LINKED ISSUE #265: Article title should be a link to open the article url
When accessing:

https://kubevirt.io/blogs/  -> titles are clickable

When clicking on communty, news, weekly updates, releases, uncathegorized or even 'year' posts:

https://kubevirt.io/blogs/news.html
https://kubevirt.io/blogs/updates.html
https://kubevirt.io/blogs/releases.html
https://kubevirt.io/blogs/uncategorized.html
https://kubevirt.io/blogs/2019/posts.html
https://kubevirt.io/blogs/2018/posts.html
https://kubevirt.io/blogs/2017/posts.html

Article titles are not clickable anymore only the 'read more' link works which is not as intuitive

### PR DESCRIPTION
**Pull request by iranzo**
**What this PR does / why we need it**:

Makes titles linkables

**Does this PR fix any issue?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
> Fixes #

Closes #265 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by codificat** [comment]
Actually I think this is
/kind enhancement

File: pages/blogs/2017/posts.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ order: 10
         {% for post in site.posts %}
           {% if post.pub-year == 2017 %}
             <li>
-              <h2 href="{{ post.url | prepend: site.baseurl }}" class="posts-title">{{ post.title }}</h2>
+              <a href="{{ post.url | prepend: site.baseurl }}"><h2 class="posts-title">{{ post.title }}</h2></a>
```
Comment (codificat) [inline pages/blogs/2017/posts.md]: Please place the `A` be inside the `H2` instead

Comment (iranzo) [reply pages/blogs/2017/posts.md]: Done

**Pull request merged**